src/components/Avatar.js

```diff
@@ -49,6 +55,16 @@ class Avatar extends PureComponent {
         };
     }
 
+    componentDidUpdate(prevProps) {
+        const isReconnecting = prevProps.network.isOffline && !this.props.network.isOffline;
+        if (
+            !this.state.imageError || !isReconnecting
+        ) {
```

Comment:
```suggestion
        if (!this.state.imageError || !isReconnecting) {
```

Do not use Prettier if you are.

Reply:
My bad, just fix it.